presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveBasicTableStatistics.java

```diff
@@ -191,15 +204,24 @@ public void testCreateExternalPartitioned()
         try {
             assertThatTableHasNoStatsFlexible(tableName);
             onPresto().executeQuery(format("CALL system.sync_partition_metadata('default', '%s', 'FULL')", tableName));
-            assertThatTableHasNoStatsFlexible(tableName);
+            // in Hive 3, syncing partitions causes stats to be populated
+            if (getHiveVersionMajor() < 3) {
+                BasicStatistics tableStatistics = getBasicStatisticsForTable(onHive(), tableName);
+                assertThatStatisticsAreNotPresent(tableStatistics);
+            }
+
+            onPresto().executeQuery(format("ANALYZE %s", tableName));
```

Comment:
remove?

We're supposedly testing CREATE TABLE that it doesn't produce bogus stats.